### PR TITLE
Feature/update css

### PR DESCRIPTION
**Pull request by hborrelli1**
Worked on css styling tweaks to fix spacing issues, as well as button layouts. 

Styling matches comp more closely, with fewer superfluous classes.

- Lili, do you think these CSS tweaks are good? 
- I replaced the "x" with an image for the close button to style easier.
- I removed the .left-guess class and replaced those styles using pseudo styles to remove unnecessary classes from the HTML.

Have any questions?

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -147,37 +160,35 @@ fieldset input {
...
 
-.current-guess {
+.card_results .flex-parent .current-guess {
   height: 100px;
 }
 
-.left-guess {
-  border-right: 2px solid #cecece;
+.card_results .flex-parent .guess-box:last-of-type {
+  border-left: 2px solid #cecece;
```
Comment (lmanriq) [inline styles.css:169]: Oh, I didn't know that you could use pseudo-elements in that way. That seems like a good way for us to keep our class names to a minimum.

File: styles.css [hunk 2]
```diff
@@ -218,19 +229,32 @@ fieldset input {
...
+
+.game-footer .time {
+  text-align:center;
+}
 
-.time {
-/* I did this to center the time, idk if that's janky */
-  margin-right: 60px;
+.game-footer .btn-wrap {
+  text-align:right;
```
Comment (lmanriq) [inline styles.css]: This spacing works better when we resize the page, good job.

File: index.html [hunk 3]
```diff
@@ -83,7 +83,9 @@ <h4 id="challenger2-name">Challenger 2 Name</h4>
           <section class="game-footer">
             <p><span class="guess-number">8</span> Guesses</p>
             <p class="time"><span class="minute">1</span> Minute <span class="second">35</span> second</p>
-            <button type="button" name="remove-box">x</button>
+            <div class="btn-wrap">
+              <button type="button" name="remove-box">x</button>
+            </div>
```
Comment (lmanriq) [inline index.html]: I think the img will make styling more straightforward, I agree.

File: styles.css (hunk 1 above)
Comment (hborrelli1) [reply styles.css:169]: Yeah, pseudo elements can be super useful!  Check out all the possibilities:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Pseudo-classes_and_pseudo-elements

**Pull request merged**